src/pages/[platform]/build-a-backend/auth/concepts/external-identity-providers/index.mdx

```diff
@@ -387,6 +387,72 @@ await signInWithRedirect({
 });
 ```
 
+### Redirect URLs
+
+For _Sign in Redirect URL(s)_ inputs, you can set one URL for local development and one for production. For example: `http://localhost:3000/` in dev and `https://www.example.com/` in production. The same is true for _Sign out Redirect URL(s)_.
+
+If you have multiple redirect URLs, you'll need to pass them in your Amplify configuration. For example:
+
+```javascript
+Amplify.configure({
+  Auth: {
+    Cognito: {
+      loginWith: {
+        oauth: {
+          redirectSignIn: [
+            'http://localhost:3000/',
+            'https://www.example.com/'
+          ],
+          redirectSignOut: [
+            'http://localhost:3000/',
+            'https://www.example.com/'
+          ],
+          ...oauthConfig
+        }
+      },
+      ...userPoolConfig
+    }
+  }
+});
+```
```

Comment:
can we show the defineAuth snippet for this? there's a few undefined values in this snippet and folks configuring auth in their backend don't typically interact with configuring the lib directly

Reply:
Since this is under "Set up your frontend", it does makes sense to show this right? defineAuth would be redundant as it's shown in above sections where we are defining the backend. 

I will remove the undefined values or update to one that is more realistic.